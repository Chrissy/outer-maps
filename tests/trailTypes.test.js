import React from "react";
import TrailTypes from "../components/trailTypes";
import renderer from "react-test-renderer";

it("renders", () => {
  const tree = renderer
    .create(<TrailTypes hike={30} bike={30} ohv={0} horse={40} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
