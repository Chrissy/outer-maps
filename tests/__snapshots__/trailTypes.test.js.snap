// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
Array [
  .emotion-8 {
  height: 0.75em;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 1em;
}

.emotion-0 {
  height: 100%;
  width: 30%;
  background-color: #5800EC;
}

.emotion-2 {
  height: 100%;
  width: 40%;
  background-color: #3232FF;
}

.emotion-4 {
  height: 100%;
  width: 30%;
  background-color: #FF9500;
}

.emotion-6 {
  height: 100%;
  width: 0%;
  background-color: #FF5500;
}

<div
    className="emotion-8 emotion-9"
  >
    <div
      className="emotion-0 emotion-1"
    />
    <div
      className="emotion-2 emotion-1"
    />
    <div
      className="emotion-4 emotion-1"
    />
    <div
      className="emotion-6 emotion-1"
    />
  </div>,
  .emotion-16 {
  display: grid;
  grid-template-columns: 0.5fr 0.5fr;
  grid-column-gap: 1em;
  grid-row-gap: 0.25em;
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 0.75em;
  color: #5800EC;
}

.emotion-0 {
  height: auto;
  width: 1.8em;
  height: 1.8em;
  margin: 0 0.25em 1em 0.25em;
}

.emotion-0 path {
  fill: #5800EC;
}

.emotion-6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 0.75em;
  color: #3232FF;
}

.emotion-4 {
  height: auto;
  width: 1.8em;
  height: 1.8em;
  margin: 0 0.25em 1em 0.25em;
}

.emotion-4 path {
  fill: #3232FF;
}

.emotion-10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 0.75em;
  color: #FF9500;
}

.emotion-8 {
  height: auto;
  width: 1.8em;
  height: 1.8em;
  margin: 0 0.25em 1em 0.25em;
}

.emotion-8 path {
  fill: #FF9500;
}

.emotion-14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 0.75em;
  color: #FF5500;
}

.emotion-12 {
  height: auto;
  width: 1.8em;
  height: 1.8em;
  margin: 0 0.25em 1em 0.25em;
}

.emotion-12 path {
  fill: #FF5500;
}

<div
    className="emotion-16 emotion-17"
  >
    <div
      className="emotion-2 emotion-3"
    >
      <test-file-stub
        className="emotion-0 emotion-1"
      />
      30%
       Hike
    </div>
    <div
      className="emotion-6 emotion-3"
    >
      <test-file-stub
        className="emotion-4 emotion-1"
      />
      40%
       Horse
    </div>
    <div
      className="emotion-10 emotion-3"
    >
      <test-file-stub
        className="emotion-8 emotion-1"
      />
      30%
       Bike
    </div>
    <div
      className="emotion-14 emotion-3"
    >
      <test-file-stub
        className="emotion-12 emotion-1"
      />
      0%
       OHV
    </div>
  </div>,
]
`;
